builders/macos-python-builder.psm1

```diff
@@ -180,7 +202,7 @@ class macOSPythonBuilder : NixPythonBuilder {
 
         $PkgVersion = [semver]"3.11.0-beta.1"
 
-        if (($this.Version -ge $PkgVersion) -or ($this.Architecture -eq "arm64")) {
+        if (($this.Version -ge $PkgVersion) -and ($this.Architecture -eq "x64"))  {
```

Comment:
arm64 always uses the universal2 installer.
most of the changes here are not required if this stays the same.